src/internal/transport/OnLog.ts

```diff
@@ -0,0 +1,111 @@
+import { logger } from "../..";
```

Comment:
I thought we didn't want to import from index.ts within the same project.

Reply:
That's there so that other people can implement a `LogHandler` without the need to import from an internal directory.